mesonbuild/mdist.py

```diff
@@ -46,7 +47,7 @@ def add_arguments(parser: argparse.ArgumentParser) -> None:
     parser.add_argument('--allow-dirty', action='store_true',
                         help='Allow even when repository contains uncommitted changes.')
     parser.add_argument('--formats', default='xztar',
-                        help='Comma separated list of archive types to create. Supports xztar (default), gztar, and zip.')
+                        help='Comma separated list of archive types to create. Supports xztar (default), bztar, gztar, and zip')
```

Comment:
Why is the period being deleted here?

Reply:
Complete accident. I'll fix it in a next push if there is a promise that this feature will be merged.

Reply:
Yes.

Reply:
Eli, last time we talked about this, you and @jpakkane didn't seem to think there was value in bz2 support.

Reply:
Ok. I share the same opinion, basically.